Update meeting times in README, make contributing section more apparent

tothambrus11 opened the pull request:
I moved the contributing section to the beginning of our readme, so that it's not hidden below pages of setup guides. I also updated the meeting times, and added a time converter, so that people all around the world can quickly see when exactly the meetings happen.

I also added the slack and zoom links to CONTRIBUTING.md, which might be helpful on that page too, though I'm not entirely sure whether the redundancy is worth it, so I can remove that if needed.

kyouko-taiga commented:
Thanks!

Do you know what happened with the markdown links?

dabrahams commented:
Others may disagree, but I don't think moving the contributing section to the top is necessarily desirable.  I don't think it's the first thing people want when they arrive at a project.  They most likely want to try Hylo out before investing any energy in the codebase.  Regardless, IMO the section should simply point at the CONTRIBUTING document, since we have one.

Lastly, @tothambrus11 please either resolve conflicts and move this PR forward so it can be merged, or close it.  Having languishing open issues and PRs is best avoided.

tothambrus11 commented:
Thank you for the feedback and ping.
I moved back the contribution section to its original place. The link link checker was this time actually right, it seems that the website I used for date conversion is not functioning anymore, so I removed the link for now...
